Packs/CommonScripts/Scripts/ExportAuditLogsToFile/ExportAuditLogsToFile.py

```diff
@@ -82,7 +82,7 @@ def main():   # pragma: no cover
 
     # if there are more events than the default size, page through and get them all
     while len(audits) < total:
-        if body.get("page"):  # pagination for xsoar-6
+        if demisto_version.startswith("6"):  # pagination for xsoar-6
```

Comment:
Jsut like [here](https://github.com/demisto/content/pull/35280/files#diff-3b0c8508c042ac2156aec60db9f17d239aa04f3b499a6efc6e7894416bcb88e9R46), `if body.get("page")` is not a googd condition because `page` is set by default to `0` which is `False`